db/migrate/20220905140742_create_books.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBooks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.string :description, null: false
+      t.float :price, null: false, precision: 5, scale: 2
```

Comment:
nit: for money values it's better to use integer type and `price_cents` column. See [money-rails](https://github.com/RubyMoney/monetize) gem